test/e2e/common_test.go

```diff
@@ -53,8 +53,8 @@ func defaultShoot(generateName string) *gardencorev1beta1.Shoot {
 			SecretBindingName: pointer.String("local"),
 			CloudProfileName:  pointer.String("local"),
 			Kubernetes: gardencorev1beta1.Kubernetes{
-				Version:                     "1.26.0",
-				EnableStaticTokenKubeconfig: pointer.Bool(true),
+				Version:                     "1.27.1",
+				EnableStaticTokenKubeconfig: pointer.Bool(false),
```

Comment:
You could remove the `enableStaticTokenKubeconfig`. It is deprecated and scheduled to be removed in Gardener v1.120. Ref https://github.com/gardener/gardener/blob/d839b336dfd7576f6c4aff67dc0981c4e20209b2/pkg/apis/core/v1beta1/types_shoot.go#L619-L624